packages/firebase_app_check/firebase_app_check/example/web/index.html

```diff
@@ -33,6 +33,9 @@
   <link rel="manifest" href="manifest.json">
 </head>
 <body>
+  <script>
+    self.FIREBASE_APPCHECK_DEBUG_TOKEN = true;
```

Comment:
Is it supposed to stay?

Reply:
yeah, just allows us to use debug token to test when we run the example app 👍